<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameIntroFrame</class>
 <widget class="QFrame" name="GameIntroFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/image/images/backgrounds/FindTrusuer_Background.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="pushBtn_Close">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>70</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>71</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/images/Main Btns/Normal Btns/Btns_Slices_52.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn_Next">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>20</y>
     <width>21</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/images/backgrounds/Background_Transparent.png);</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>20</y>
     <width>592</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/images/backgrounds/Background_Transparent.png);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget1" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget2" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushBtn_Prev">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>20</y>
     <width>31</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/images/backgrounds/Background_Transparent.png);</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
